server/api/books/index.ts

```
@@ -0,0 +1,144 @@
+import { PrismaClient } from "@prisma/client";
+import { defineEventHandler, readBody,createError } from "h3";
+import {v4 as uuidv4} from "uuid";
+import { BOOK_TYPES,STATUS_TYPES } from "~/constants/Types";
+
+declare global {
+  var prisma: PrismaClient | undefined;
+}
+const prisma = global.prisma || new PrismaClient();
```

Biome (1.9.4)

[error] Shouldn't redeclare 'prisma'. Consider to delete it or rename it.

'prisma' is defined here:


(lint/suspicious/noRedeclare)